packages/subgraph/schema.graphql

```diff
@@ -104,7 +104,7 @@ interface TokenTransfer {
 type ERC20Transfer implements TokenTransfer @entity(immutable: true) {
   id: ID!
   dao: Dao!
-  token: ERC20Contract!
+  token: Token!
```

Comment:
This shouldn't affect ths SDK as long as as it contains this

```
id
name
symbol
decimals
```
I believe it does so no problem with that